regexes/client/libraries.yml

```diff
@@ -566,3 +566,13 @@
   name: 'request'
   version: ''
   url: 'https://github.com/request/request'
+
+- regex: 'kiwi-tcms/(\d+[\.\d]+)'
+  name: 'Kiwi TCMS'
+  version: '$1'
+  url: 'https://kiwitcms.org'
+
+- regex: 'tcms-api/(\d+[\.\d]+)'
+  name: 'Kiwi TCMS API'
+  version: '$1'
+  url: 'https://kiwitcms.org'
```

Comment:
Also, can you move the regexes for libraries at the top of the file? This is to avoid the user-agent with `Java` being detected as `Java` library.